AnnoyancesFilter/sections/widgets.txt

```diff
@@ -552,6 +552,7 @@ subbota.ua##aside.sidebar > div[id^="custom_html-"]
 !
 ! Donation widgets (which are really annoying only)
 !
+m.hani.co.kr##.bn-promotion-support
```

Comment:
According to their mobile and desktop versions, I'd leave these rules.
```
hani.co.kr##.bn-promotion-support
m.hani.co.kr##.support-content
```
These is also a share https://www.hani.co.kr/arti/opinion/because/994556.html
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/124557940-c046b800-de42-11eb-848d-4554765466f5.png)

</details><br/>
These rules should be ok

```
hani.co.kr##.article-tools > ul.tools > li[onclick*="popupHaniWindow"][onclick*="share"]
hani.co.kr##.article-tools > ul.tools > li[data-share-url]
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/87557/commits/fc3228c3ecc07cee9adf0b8ed82d978958144aa3
Is it okay?